components/molecule/notification/test/index.test.js

```diff
@@ -40,11 +42,13 @@ describe(json.name, () => {
     // When
     const {
       POSITION,
+      ALIGN_ITEMS,
       AUTO_CLOSE,
       TYPES,
       VARIATIONS,
       BRDS_SIZE,
       moleculeNotificationPosition,
+      moleculeNotificationAlignItems,
```

Comment:
Add a test description  `moleculeNotificationAlignItems` section at the end of the file just like `moleculeNotificationPosition` with its values and type